terraform_files/terraform.tfvars.json

```diff
@@ -2,20 +2,25 @@
   "cluster_name": "test-infra-cluster",
   "cluster_domain": "redhat.com",
   "machine_cidr": "192.168.126.0/24",
+  "provisioning_cidr": "192.168.140.0/24",
   "master_count": "3",
   "worker_count": "1",
   "image_path": "tmp",
   "libvirt_uri": "qemu+tcp://192.168.122.1/system",
   "libvirt_network_if": "tt0",
+  "libvirt_network_if_2": "tt1",
```

Comment:
maybe stt1? it will not break you with index?

Reply:
fixed